test/cmdlineTests/transient_storage_layout_smoke_empty/input.json

```diff
@@ -0,0 +1,16 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {
+			"content": "//SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\ncontract A { }"
+		}
+	},
+	"settings": {
+		"outputSelection": {
+			"fileA": {
+				"A": [ "transientStorageLayout" ],
+				"": [ "transientStorageLayout" ]
```

Comment:
We don't have file-level transient storage layout output, so requesting it is pointless here (unless you're specifically testing that it does nothing, but that does not seem very useful).

It's also done in some of the other tests.

Reply:
```suggestion
				"A": ["transientStorageLayout"],
				"": ["transientStorageLayout"]
```